src/unix/bsd/freebsdlike/freebsd/mod.rs

```diff
@@ -4788,6 +4793,14 @@ safe_f! {
     pub {const} fn PR_SCTP_VALID_POLICY(x: c_int) -> bool {
         PR_SCTP_POLICY(x) <= SCTP_PR_SCTP_MAX
     }
+
+    pub {const} fn PPROT_OP(o: c_int) -> c_int {
```

Comment:
I _personally_ would prefer the PPROT changes in a separate commit (same PR is ok), so they can get a more focused commit message (perhaps with a reference link as well)

Reply:
but ultimately I will have to squash it so..